### ISSUE TITLE
 No such file or directory?

### ISSUE THREAD

**Issue by icehongssii** [bwilcox-1234/ChatScript#219]
Hi, I am really new to ChatScript. i was about to run ChatScript in the directory 'Binaries'
`C:\Users\iceho\Desktop\ChatScript\BINARIES>ChatScript`
But, the error message keeps coming.
`  in cs_init.txt at 0:
    Error opening utf8writeappend file LOGS/startlog.txt: No such file or directory
`
thanks for your willing to help.
I hope to get the answer.

**Comment by rmsmith007** [comment]
As a workaround, create a LOGS directory within the ChatScript directory.

**Comment by icehongssii** [comment]
@rmsmith007 I just bow to where you are. Thanks

**Comment by yaleimeng** [comment]
this is the issue I asked 。same as #203 
And this Project has moved to [ChatScriptNLP](https://github.com/chatscriptnlp/ChatScriptNLP)。

**Comment by lookdeepu** [comment]
@rmsmith007 You are awesome. Thank you.

**Comment by MaffooBristol** [comment]
Oh what. The project has been moved!? Why on earth is that not even mentioned anywhere on this project?

**Comment by rmsmith007** [comment]
It's mentioned [here](https://www.chatbots.org/ai_zone/viewthread/3325/) that Bruce Wilcox has lost the credentials for the old project (i.e. this one).

**Comment by MaffooBristol** [comment]
Yes, I've commented on that, and am having long discussions regarding it on the new project
